autobuild/examples/xquery-gethINT-avaxINT.yaml

```diff
@@ -0,0 +1,72 @@
+# EXAMPLE INPUT FILE FOR XQUERY WITH EXTERNAL GETH AND INTERNAL AVAX
+--- # snode daemon LIST
+- j2template: dockercompose.j2
+  name: dockercompose
+  daemons:
+    # DO NOT DELETE XR_PROXY ENTRY
+    - name: XR_PROXY # DO NOT DELETE
+      image: blocknetdx/exrproxy:0.8.0 # DO NOT DELETE
+      config_mount_dir: /snode/xr_proxy/config # DO NOT DELETE
+      nginx_mount_dir: /snode/xr_proxy/nginx # DO NOT DELETE
+    # DO NOT DELETE ETH ENTRY
+    - name: ETH # DO NOT DELETE
+      image: ethereum/client-go:latest
+      geth_data_mount_dir: /snode # DO NOT DELETE    
+    - name: PAYMENT # DO NOT DELETE
+      payment_tier1: 35
+      payment_tier2: 200
+      image: blocknetdx/eth-payment-processor:0.5.2 # DO NOT DELETE
+      postgresql_data_mount_dir: /snode/eth_pymt_db # DO NOT DELETE
+    # DO NOT DELETE SNODE ENTRY
+    - name: SNODE # DO NOT DELETE
+      image: blocknetdx/servicenode:latest # DO NOT DELETE
+      config_mount_dir: /snode # DO NOT DELETE
+      data_mount_dir: /snode # DO NOT DELETE
+    - name: BTC
+      image: blocknetdx/bitcoin:v0.20.0
+      config_mount_dir: /snode
+      data_mount_dir: /snode
+    - name: LTC
+      image: blocknetdx/litecoin:v0.18.1
+      config_mount_dir: /snode
+      data_mount_dir: /snode
+    - name: DGB
+      image: blocknetdx/digibyte:v7.17.2
+      config_mount_dir: /snode
+      data_mount_dir: /snode
+    - name: SYS
+      image: blocknetdx/syscoin4:v4.3.0
+      config_mount_dir: /snode
+      data_mount_dir: /snode
+    - name: PIVX
+      image: blocknetdx/pivx:v5.3.2
+      config_mount_dir: /snode
+      data_mount_dir: /snode      
+    - name: AVAX 
+      image: avaplatform/avalanchego:v1.7.2
```

Comment:
Might as well use the latest avalanchego image: `avaplatform/avalanchego:v1.7.3`.
I've been running it for a couple days on another VM. It seems to work fine.

Reply:
sorry, my bad. the last commit was regarding #75.

Reply:
Sorry, didn't understand how your comment there about #75 relates to my request here. My request is simply to replace this line:
```
      image: avaplatform/avalanchego:v1.7.2
```
with this line:
```
      image: avaplatform/avalanchego:v1.7.3
```